index.html

```diff
@@ -93,7 +94,7 @@
 User agents MUST ignore extraneous characters found after a `metric-name` but before the first `server-timing-param` and before the next `server-timing-metric`.
 
 <p data-link-for="PerformanceServerTiming">
-  This specification establishes the server-timing-params for server-timing-param-names <a data-lt="PerformanceServerTiming.duration">dur</a> and <a data-lt="PerformanceServerTiming.description">desc</a>, both optional.
+  This specification establishes the server-timing-params for server-timing-param-names {{duration}} and {{description}}, both optional.
```

Comment:
This should probably be:
```suggestion
  This specification establishes the server-timing-params for server-timing-param-names "dur" for {{duration}} and "desc" for {{description}}, both optional.
```

Reply:
agreed